web/server/build.js

```diff
@@ -0,0 +1,14 @@
+const esbuild = require("esbuild");
+const { glob } = require("glob");
+
+// Builds the server code into CJS using esbuild. The package.json file is CJS (i.e. not type=module)
+// because setting to module will cause grpc-web code to break.
+//
+// In addition, our esbuild is done via the JavaScript API so that we can use glob syntax
+(async () => {
+  let result = await esbuild.build({
+    entryPoints: await glob(["server/**/*.ts", "gen/**/*.ts"]),
+    outdir: "dist",
+    format: "cjs",
+  });
+})();
```

Comment:
This seems complicated. Can we not simply start the server with `npx tsx ...`?

Reply:
I tried that at first and ran into the `incremental` breaking change with esbuild v0.17 and originally decided to punt on it until I could remove Karma. But, point taken. I pinned esbuild to v0.16.17 instead and removed all the complications around building.  Still want to remove Karma, but this is a much better interim solution.